modules/aws-velero/variables.tf

```diff
@@ -0,0 +1,35 @@
+variable "bucket_name" {
+  type = string
+}
+
+variable "s3_bucket_arn" {
+  type = string
+}
+
+variable "k8s_cluster_oidc_arn" {
+  type = string
+}
+
+variable "region_name" {
+  type = string
+}
```

Comment:
Isn't this support to represent backup region? Could you rename variable to clarify?

Reply:
It has been renamed to `bucket_region` to be more aligned to `bucket_name`